po/common/common.pot

```diff
@@ -4811,11 +4811,15 @@ msgctxt "attribute_ecoscore_setting_note"
 msgid "The Eco-Score is an environmental score (ecoscore) from A to E which makes it easy to compare the impact of food products on the environment."
 msgstr ""
 
-# keep %s, it will be replaced by the letter A, B, C, D or E
+# keep %s, it will be replaced by the letter A+, A, B, C, D, E or F
 msgctxt "attribute_ecoscore_grade_title"
 msgid "Eco-Score %s"
 msgstr "Eco-Score %s"
 
+msgctxt "attribute_ecoscore_a_plus_description_short"
+msgid "Very low environmental impact"
```

Comment:
GetText will complain about duplicate ids like, and Crowdin will get confused and cross contaminate between the 2 strings